src/Microsoft.Kusto.ServiceLayer/DataSource/DataSourceBase.cs

```diff
@@ -87,17 +79,6 @@ public async Task<T> ExecuteScalarQueryAsync<T>(string query, CancellationToken
         /// <inheritdoc/>
         public abstract void UpdateDatabase(string databaseName);
 
```

Comment:
These 4 were moved to KustoIntellisenseHelper since they're related to that and didn't need to be on KustoDataSource